app/main.py

```diff
@@ -1,8 +1,51 @@
+def limitation(value: float,
+               min_value: float,
+               max_value: float) -> float:
+    return max(min(value, max_value), min_value)
```

Comment:
The best approach of placing parameters is to move them on the next line after function name